package.json

```diff
@@ -21,11 +21,12 @@
   },
   "license": "BSD-2-Clause",
   "scripts": {
-    "prepare": "npm run update-version && npm run build",
```

Comment:
We should now add the build step to [lint job](https://github.com/eslint/eslint-scope/blob/82a7e6d9de8f4fca48e99779e9573dd46adbc18c/.github/workflows/ci.yml#L10-L22). Otherwise, it will fail on `node/no-missing-require` rule as there is no `dist/eslint-scope.cjs` which is required from a test file.

This isn't yet observable on this PR because we just switched to `main` but `ci.yml` hasn't been updated.

Reply:
Good catch! The CI was broken otherwise this would have been obvious. I'll fix that too.